mcmd/commands/make.py

```diff
@@ -97,15 +117,38 @@ def _get_user_group_membership(user: str, group_name: str) -> Optional[Membershi
 
 
 def _get_memberships_for_user(user_name: str) -> List[Membership]:
-    # Don't use a query here because the indexer might not've finished yet
```

Comment:
This sounds ominous

Reply:
Yes it does, doesn't it 😅 

I remember writing this after I had a timing issue with a script like:
```
add group test
make henk test_manager
```

Now I just wasn't sure if that made sense so I tried it a couple of times and couldn't reproduce it.

Reply:
In an ideal world each command would validate its own success, I think. Working around timing issues by avoiding queries didn't feel right to me, so I removed it.